### PR TITLE
Preview alerts

### PR DESCRIPTION
**Pull request by jamarw**
1. Added alert-styles.css to style blockquotes.
2. Added alert-styles.js to set alert classes and structure content.
3. Remove unused css and js files.
4. Update package.json to use custom css and js.

https://github.com/Microsoft/vscode-docs-authoring/issues/200

### CODE REVIEW

File: docs-preview/media/alert-styles.css [hunk 1]
```diff
@@ -1,16 +1,74 @@
...
-	background-position: top left;
-	background-repeat: no-repeat;
-	text-indent: 23px;
-	}
-	blockquote.NOTE span {
-		  display: block;
-		  background-repeat: no-repeat;
-		  background-position: bottom right;
-		}
+  font: 14px/20px italic Times, serif;
```
Comment (adunndevster) [inline docs-preview/media/alert-styles.css]: It looks like the only part of these blockquote styles that are not shared are the background attributes?

Maybe you can make a common blockquote.alert class with the shared values, and then keep the other items separate?

File: docs-preview/media/alert-styles.css [hunk 2]
```diff
@@ -1,16 +1,74 @@
...
+blockquote.TIP {
+  font: 14px/20px italic Times, serif;
+  padding: 8px;
+  background: rgba(0, 100, 73, 0.89);
+  margin: 5px;
+  background-position: top left;
+  background-repeat: no-repeat;
+  text-indent: 23px;
+}
+blockquote.TIP span {
```
Comment (adunndevster) [inline docs-preview/media/alert-styles.css]: also all of the spans look the same, so maybe you could just assign them to blockquote.alert span?

File: docs-preview/media/alert-styles.js [hunk 3]
```diff
@@ -1,19 +1,19 @@
-var blockquoteTags = document.getElementsByTagName('blockquote');
+var blockquoteTags = document.getElementsByTagName("blockquote");
```
Comment (adunndevster) [inline docs-preview/media/alert-styles.js]: Just an idea... but maybe you could make an array filled with strings of the different kinds of alerts, and then map a function onto the array to check for each kind, and assign its corresponding attribute?

Let me try working on an example...

Comment (adunndevster) [reply docs-preview/media/alert-styles.js]: @jamarw  - what about something like this?

`var blockquoteTags = document.getElementsByTagName("blockquote");
var alerts = ["NOTE", "TIP", "IMPORTANT", "CAUTION", "WARNING"];

alerts.map(aa => {
  for (var i = 0; i < blockquoteTags.length; i++) {
    let tag = blockquoteTags[i];
    if (tag.innerHTML.toUpperCase().indexOf(`[!${aa}]`) > -1) {
      tag.setAttribute("class", aa);
    }
  }
});`

Comment (jamarw) [reply docs-preview/media/alert-styles.js]: @adunndevster Great idea!  I looked at a few options similar to this and didn't come up with anything as clean.  Thx

**Pull request merged**